app/views/caseload.html

```diff
@@ -12,133 +12,219 @@
+            <td headers="{{caseType.displayName}}_om {{caseType.displayName}}_grade">{{ member.grades[i].grade }}</td>            
+            <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_overall" >{{ member.grades[i].totalCases | round(1) }}%</td>              
+            <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_untiered">{{ member.grades[i].untiered | round(1) }}%</td>
+            <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_d2">{{ member.grades[i].d2 | round(1) }}%</td>
+            <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_d1">{{ member.grades[i].d1 | round(1) }}%</td>
+            <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_c2">{{ member.grades[i].c2 | round(1) }}%</td>
+            <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_c1">{{ member.grades[i].c1 | round(1) }}%</td>
+            <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_b2">{{ member.grades[i].b2 | round(1) }}%</td>
+            <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_b1">{{ member.grades[i].b1 | round(1) }}%</td>
+            <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_a">{{ member.grades[i].a | round(1) }}%</td>
           </tr>
-        </thead>
-
-        <tbody class="">
-          {% for member in caseType.array %}
-            <tr>
-              <td headers="{{caseType.displayName}}_om {{caseType.displayName}}_name"><a href="/{{ childOrganisationLevel }}/{{ member.linkId }}">{{ member.name }}</a></td>
-              <td headers="{{caseType.displayName}}_om {{caseType.displayName}}_grade">{{ member.gradeCode }}</td>
-              <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_untiered">{{ member.untiered }}</td>
-              <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_d2">{{ member.d2 }}</td>
-              <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_d1">{{ member.d1 }}</td>
-              <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_c2">{{ member.c2 }}</td>
-              <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_c1">{{ member.c1 }}</td>
-              <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_b2">{{ member.b2 }}</td>
-              <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_b1">{{ member.b1 }}</td>
-              <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_a">{{ member.a }}</td>
-              <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_total">{{ member.totalCases }}</td>
-            </tr>
           {% endfor %}
-        </tbody>
-      </table>
-    </div>
-
+        {% endfor %}
+      </tbody>
+    </table>
+  {% endif %}
   {% endfor %}
-
-  </div>
 </div>
-
-{% elseif organisationLevel !== 'team'%}
```

Comment:
If we don't have caseload for individual (OM), is it worth removing the `Caseload` tab all together, that way the tabs will be aligned including the reductions.

Reply:
Yeah, removed caseload for OM